.circleci/config.yml

```diff
@@ -60,6 +60,7 @@ jobs:
       - restore_cache_checkout
       - run_yarn
       - run: yarn ci-check
+      - run: export NODE_OPTIONS=--max_old_space_size=4096
```

Comment:
A site build requires more than default node memory.

Deploy builds the site and use 4go. But tests workflow also build the site, so it should rather use 4go as well.